lms/djangoapps/instructor/views/api.py

```diff
@@ -1044,16 +1043,12 @@ def get_coupon_codes(request, course_id):  # pylint: disable=unused-argument
     Respond with csv which contains a summary of all Active Coupons.
     """
     course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
-    active_coupons = Coupon.objects.filter(
-        Q(course_id=course_id),
-        Q(is_active=True),
-        Q(expiration_date__gt=datetime.datetime.now(pytz.UTC)) |
-        Q(expiration_date__isnull=True)
-    )
+    coupons = Coupon.objects.filter(course_id=course_id)
```

Comment:
@chrisndodge  i have removed the Q object from here, since we are not encapsulating the collection of keywod arguments(Field Lookups) anymore.